<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Phylogenetic Clustering (Phyloclustering)</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Phylogenetic Clustering (Phyloclustering)
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;phyclust&rsquo; version 0.1-18</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../demo">Code demos</a>.  Use <a href="../../utils/help/demo">demo()</a> to run them.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#W">W</a>
<a href="#misc">misc</a>
</p>

<table width="100%">
<tr><td style="width: 25%;"><a href="00_phyclust-package.html">phyclust-package</a></td>
<td>Phyloclustering - Phylogenetic Clustering</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="nid.aid.cid.html">acode2aid</a></td>
<td>Transfer nids (0, 1, ..., 4) , aids (0, 1, ..., 21) and cids (0, 1, ..., 64)</td></tr>
<tr><td style="width: 25%;"><a href="nid.aid.cid.html">aid2acode</a></td>
<td>Transfer nids (0, 1, ..., 4) , aids (0, 1, ..., 21) and cids (0, 1, ..., 64)</td></tr>
<tr><td style="width: 25%;"><a href="as.star.tree.html">as.star.tree</a></td>
<td>Coerce a Rooted Tree to a Star Tree in Class phylo</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="bootstrap.seq.html">bootstrap.seq</a></td>
<td>Bootstrap Sequences from a Fitted Model and Star Tree.</td></tr>
<tr><td style="width: 25%;"><a href="bootstrap.seq.data.html">bootstrap.seq.data</a></td>
<td>Bootstrap a seq.data from a Fitted Model.</td></tr>
<tr><td style="width: 25%;"><a href="bootstrap.star.trees.html">bootstrap.star.trees</a></td>
<td>Bootstrap a Star Tree from a Fitted Model.</td></tr>
<tr><td style="width: 25%;"><a href="bootstrap.star.trees.seq.html">bootstrap.star.trees.seq</a></td>
<td>Bootstrap Sequences from a Fitted Model.</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="nid.aid.cid.html">cid2aid</a></td>
<td>Transfer nids (0, 1, ..., 4) , aids (0, 1, ..., 21) and cids (0, 1, ..., 64)</td></tr>
<tr><td style="width: 25%;"><a href="paml.baseml.html">Class baseml</a></td>
<td>Phylogenetic Analysis by Maximum Likelihood for Nucleotide Sequences</td></tr>
<tr><td style="width: 25%;"><a href="ms.html">Class ms</a></td>
<td>Generating Samples under a Wright-Fisher Neutral Model of Genetic Variation</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.html">Class phyclust</a></td>
<td>The Main Function of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="as.star.tree.html">Class phylo</a></td>
<td>Coerce a Rooted Tree to a Star Tree in Class phylo</td></tr>
<tr><td style="width: 25%;"><a href="seq.data.html">Class seq.data</a></td>
<td>A Toy Dataset in Class seq.data</td></tr>
<tr><td style="width: 25%;"><a href="seqgen.html">Class seqgen</a></td>
<td>Seq-Gen</td></tr>
<tr><td style="width: 25%;"><a href="code2nid.html">code2nid</a></td>
<td>Transfer Codes (A, G, C, T, -) and nids (0, 1, 2, 3, 4)</td></tr>
<tr><td style="width: 25%;"><a href="code2snp.html">code2snp</a></td>
<td>Transfer Nucleotide Codes / nids and SNPs / sids</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="data.phylip.crohn.html">Data Crohn's Disease</a></td>
<td>Crohn's Disease SNP Dataset in the phylip Format</td></tr>
<tr><td style="width: 25%;"><a href="data.phylip.pony.html">Data Pony 524</a></td>
<td>Great Pony 524 EIAV rev Dataset in the phylip Format</td></tr>
<tr><td style="width: 25%;"><a href="data.fasta.pony.html">Data Pony 625</a></td>
<td>Great Pony 625 EIAV rev Dataset in the Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="seq.data.html">Data Toy</a></td>
<td>A Toy Dataset in Class seq.data</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="find.best.html">find.best</a></td>
<td>Find the Best Solution of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="find.consensus.html">find.consensus</a></td>
<td>Find the Consensus Sequence</td></tr>
<tr><td style="width: 25%;"><a href="data.fasta.pony.html">Format fasta</a></td>
<td>Great Pony 625 EIAV rev Dataset in the Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="data.phylip.pony.html">Format phylip</a></td>
<td>Great Pony 524 EIAV rev Dataset in the phylip Format</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="gen.equal.star.anc.dec.html">gen.equal.star.anc.dec</a></td>
<td>Generate Comprehensive Trees.</td></tr>
<tr><td style="width: 25%;"><a href="gen.seq.html">gen.seq.HKY</a></td>
<td>Generate Sequences Given a Rooted Tree.</td></tr>
<tr><td style="width: 25%;"><a href="gen.seq.html">gen.seq.SNP</a></td>
<td>Generate Sequences Given a Rooted Tree.</td></tr>
<tr><td style="width: 25%;"><a href="gen.star.tree.html">gen.star.tree</a></td>
<td>Generate a Rooted Tree with a Star Shape</td></tr>
<tr><td style="width: 25%;"><a href="gen.unit.K.html">gen.unit.K</a></td>
<td>Generate Comprehensive Trees.</td></tr>
<tr><td style="width: 25%;"><a href="get.rooted.tree.height.html">get.rooted.tree.height</a></td>
<td>Get a Rooted Tree Height</td></tr>
<tr><td style="width: 25%;"><a href="getcut.fun.html">getcut.fun</a></td>
<td>Tzeng's Method: Finding the Best Number of Clusters</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="haplo.post.prob.html">haplo.post.prob</a></td>
<td>Tzeng's Method: Haplotype Grouping for SNP Sequences</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="ms.html">ms</a></td>
<td>Generating Samples under a Wright-Fisher Neutral Model of Genetic Variation</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="nid.aid.cid.html">nid2aid</a></td>
<td>Transfer nids (0, 1, ..., 4) , aids (0, 1, ..., 21) and cids (0, 1, ..., 64)</td></tr>
<tr><td style="width: 25%;"><a href="nid.aid.cid.html">nid2cid</a></td>
<td>Transfer nids (0, 1, ..., 4) , aids (0, 1, ..., 21) and cids (0, 1, ..., 64)</td></tr>
<tr><td style="width: 25%;"><a href="code2nid.html">nid2code</a></td>
<td>Transfer Codes (A, G, C, T, -) and nids (0, 1, 2, 3, 4)</td></tr>
<tr><td style="width: 25%;"><a href="code2snp.html">nid2sid</a></td>
<td>Transfer Nucleotide Codes / nids and SNPs / sids</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="paml.baseml.html">paml.baseml</a></td>
<td>Phylogenetic Analysis by Maximum Likelihood for Nucleotide Sequences</td></tr>
<tr><td style="width: 25%;"><a href="paml.baseml.html">paml.baseml.control</a></td>
<td>Phylogenetic Analysis by Maximum Likelihood for Nucleotide Sequences</td></tr>
<tr><td style="width: 25%;"><a href="paml.baseml.html">paml.baseml.show.default</a></td>
<td>Phylogenetic Analysis by Maximum Likelihood for Nucleotide Sequences</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.html">phyclust</a></td>
<td>The Main Function of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.e.step.html">phyclust.e.step</a></td>
<td>One E-Step of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.edist.html">phyclust.edist</a></td>
<td>Evolution Distance of Sequences</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.em.step.html">phyclust.em.step</a></td>
<td>One EM-step of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.logL.html">phyclust.logL</a></td>
<td>Log-Likelihood of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.m.step.html">phyclust.m.step</a></td>
<td>One M-Step of phyclust</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.Pt.html">phyclust.Pt</a></td>
<td>Transition Probabilities of phyclust Given Time</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.se.html">phyclust.se</a></td>
<td>The Main Function of phyclust for Sequencing Error Models</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.se.update.html">phyclust.se.update</a></td>
<td>Update phyclust Results by the Sequencing Error Model</td></tr>
<tr><td style="width: 25%;"><a href="phyclust.update.html">phyclust.update</a></td>
<td>Update phyclust Results</td></tr>
<tr><td style="width: 25%;"><a href="plotdots.html">plotdots</a></td>
<td>Dots Plots of Sequences for Visual Comparisons</td></tr>
<tr><td style="width: 25%;"><a href="plotgaps.html">plotgaps</a></td>
<td>Gaps Plots of Sequences for Visual Comparisons</td></tr>
<tr><td style="width: 25%;"><a href="plothist.html">plothist</a></td>
<td>Plot Histogram to Compare Number of Mutations.</td></tr>
<tr><td style="width: 25%;"><a href="plotnj.html">plotnj</a></td>
<td>Plot an Unrooted Trees.</td></tr>
<tr><td style="width: 25%;"><a href="plotstruct.html">plotstruct</a></td>
<td>Struct Plots of Observations Based on Posterior Probabilities</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.baseml</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.ms</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.phyclust</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.Pt</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.RRand</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.seq.data</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">print.seqgen</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
<tr><td style="width: 25%;"><a href="prune.Mu.html">prune.Mu</a></td>
<td>Prune the Center Sequences Mu</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="file.read.html">read.fasta</a></td>
<td>Read Data from Files by Formats and Return a seq.data Object</td></tr>
<tr><td style="width: 25%;"><a href="file.read.html">read.fasta.format</a></td>
<td>Read Data from Files by Formats and Return a seq.data Object</td></tr>
<tr><td style="width: 25%;"><a href="file.read.html">read.phylip</a></td>
<td>Read Data from Files by Formats and Return a seq.data Object</td></tr>
<tr><td style="width: 25%;"><a href="file.read.html">read.phylip.format</a></td>
<td>Read Data from Files by Formats and Return a seq.data Object</td></tr>
<tr><td style="width: 25%;"><a href="read.seqgen.html">read.seqgen</a></td>
<td>Read seqgen's Results and Return a seq.data</td></tr>
<tr><td style="width: 25%;"><a href="rescale.rooted.tree.html">rescale.rooted.tree</a></td>
<td>Rescale a Rooted Tree's Height</td></tr>
<tr><td style="width: 25%;"><a href="RRand.html">RRand</a></td>
<td>Rand Index and Adjusted Rand Index</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="seq.data.html">seq.data.gap</a></td>
<td>A Toy Dataset in Class seq.data</td></tr>
<tr><td style="width: 25%;"><a href="seq.data.html">seq.data.toy</a></td>
<td>A Toy Dataset in Class seq.data</td></tr>
<tr><td style="width: 25%;"><a href="seqgen.html">seqgen</a></td>
<td>Seq-Gen</td></tr>
<tr><td style="width: 25%;"><a href="code2snp.html">sid2nid</a></td>
<td>Transfer Nucleotide Codes / nids and SNPs / sids</td></tr>
<tr><td style="width: 25%;"><a href="snp2sid.html">sid2snp</a></td>
<td>Transfer SNP codes (1, 2, -) and sids (0, 1, 2)</td></tr>
<tr><td style="width: 25%;"><a href="code2snp.html">snp2code</a></td>
<td>Transfer Nucleotide Codes / nids and SNPs / sids</td></tr>
<tr><td style="width: 25%;"><a href="snp2sid.html">snp2sid</a></td>
<td>Transfer SNP codes (1, 2, -) and sids (0, 1, 2)</td></tr>
<tr><td style="width: 25%;"><a href="print.summary.html">summary.phyclust</a></td>
<td>Functions for Printing or Summarizing Objects According to Classes</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="file.write.html">write.fasta</a></td>
<td>Write Data to Files by Formats</td></tr>
<tr><td style="width: 25%;"><a href="file.write.html">write.fasta.format</a></td>
<td>Write Data to Files by Formats</td></tr>
<tr><td style="width: 25%;"><a href="file.write.html">write.paml</a></td>
<td>Write Data to Files by Formats</td></tr>
<tr><td style="width: 25%;"><a href="file.write.html">write.paml.format</a></td>
<td>Write Data to Files by Formats</td></tr>
<tr><td style="width: 25%;"><a href="file.write.html">write.phylip</a></td>
<td>Write Data to Files by Formats</td></tr>
<tr><td style="width: 25%;"><a href="file.write.html">write.phylip.format</a></td>
<td>Write Data to Files by Formats</td></tr>
</table>

<h2><a name="misc">-- misc --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="11_d.standard.code.html">.amino.acid</a></td>
<td>Standard Codes and ids for Nucleotides, SNPs, Codon, Amino Acid and Genetic Code</td></tr>
<tr><td style="width: 25%;"><a href="11_d.boundary.method.html">.boundary.method</a></td>
<td>Boundary Methods for Population Proportions</td></tr>
<tr><td style="width: 25%;"><a href="11_d.code.type.html">.code.type</a></td>
<td>Code Types of Dataset and Substitution Models</td></tr>
<tr><td style="width: 25%;"><a href="11_d.standard.code.html">.codon</a></td>
<td>Standard Codes and ids for Nucleotides, SNPs, Codon, Amino Acid and Genetic Code</td></tr>
<tr><td style="width: 25%;"><a href="11_d.Color.html">.Color</a></td>
<td>Colors for Identifying Clusters in Plots</td></tr>
<tr><td style="width: 25%;"><a href="11_d.edist.model.html">.edist.model</a></td>
<td>Evolution Distance Model</td></tr>
<tr><td style="width: 25%;"><a href="11_d.em.method.html">.em.method</a></td>
<td>EM Methods and Algorithms</td></tr>
<tr><td style="width: 25%;"><a href="11_d.EMC.html">.EMC</a></td>
<td>EM Control</td></tr>
<tr><td style="width: 25%;"><a href="11_d.EMControl.html">.EMControl</a></td>
<td>EM Control Generator</td></tr>
<tr><td style="width: 25%;"><a href="11_d.standard.code.html">.genetic.code</a></td>
<td>Standard Codes and ids for Nucleotides, SNPs, Codon, Amino Acid and Genetic Code</td></tr>
<tr><td style="width: 25%;"><a href="11_d.identifier.html">.identifier</a></td>
<td>Identifiers for Evolution Models</td></tr>
<tr><td style="width: 25%;"><a href="11_d.init.method.html">.init.method</a></td>
<td>Initialization Methods for EM Algorithms</td></tr>
<tr><td style="width: 25%;"><a href="11_d.init.procedure.html">.init.procedure</a></td>
<td>Initialization Procedures for EM Algorithms</td></tr>
<tr><td style="width: 25%;"><a href="11_d.label.method.html">.label.method</a></td>
<td>Label Method</td></tr>
<tr><td style="width: 25%;"><a href="11_d.standard.code.html">.missing.code</a></td>
<td>Standard Codes and ids for Nucleotides, SNPs, Codon, Amino Acid and Genetic Code</td></tr>
<tr><td style="width: 25%;"><a href="11_d.standard.code.html">.nucleotide</a></td>
<td>Standard Codes and ids for Nucleotides, SNPs, Codon, Amino Acid and Genetic Code</td></tr>
<tr><td style="width: 25%;"><a href="11_d.se.model.html">.se.model</a></td>
<td>Sequencing Error Model</td></tr>
<tr><td style="width: 25%;"><a href="11_d.show.option.html">.show.option</a></td>
<td>Show Available Options</td></tr>
<tr><td style="width: 25%;"><a href="11_d.standard.code.html">.snp</a></td>
<td>Standard Codes and ids for Nucleotides, SNPs, Codon, Amino Acid and Genetic Code</td></tr>
<tr><td style="width: 25%;"><a href="11_d.substitution.html">.substitution.model</a></td>
<td>Substitution Models for Mutation Processes</td></tr>
</table>
</body></html>
