<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>R: Read seqgen's Results and Return a seq.data</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>

<table width="100%" summary="page for read.seqgen {phyclust}"><tr><td>read.seqgen {phyclust}</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>Read seqgen's Results and Return a seq.data</h2>

<h3>Description</h3>

<p>This function can read the results generated by <code>seqgen</code> and
turn into a object in <code>Class seq.data</code>.
</p>


<h3>Usage</h3>

<pre>
read.seqgen(text, byrow = TRUE, code.type = .code.type[1])
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>text</code></td>
<td>
<p>a text vector generated by <code>seqgen</code>.</p>
</td></tr>
<tr valign="top"><td><code>byrow</code></td>
<td>
<p>advanced option, default = TRUE.</p>
</td></tr>
<tr valign="top"><td><code>code.type</code></td>
<td>
<p>either &quot;NUCLEOTIDE&quot; (default) or &quot;SNP&quot;.</p>
</td></tr>
</table>


<h3>Details</h3>

<p>If <code>code.type</code> is &quot;SNP&quot;, the A, G will be transfered to 1, and
the C, T will be transfered to 2.
</p>


<h3>Value</h3>

<p>Return an object in <code>Class seq.data</code>.
</p>


<h3>Author(s)</h3>

<p>Wei-Chen Chen <a href="mailto:wccsnow@gmail.com">wccsnow@gmail.com</a>
</p>


<h3>References</h3>

<p>Phylogenetic Clustering Website:
<a href="http://snoweye.github.io/phyclust/">http://snoweye.github.io/phyclust/</a>
</p>


<h3>See Also</h3>

<p><code><a href="seqgen.html">seqgen</a></code>,
<code><a href="gen.seq.html">gen.seq.HKY</a></code>,
<code><a href="gen.seq.html">gen.seq.SNP</a></code>.
</p>


<h3>Examples</h3>

<pre>
## Not run: 
library(phyclust, quiet = TRUE)

set.seed(123)
ret.ms &lt;- ms(nsam = 5, nreps = 1, opts = "-T")
ret.seqgen &lt;- seqgen(opts = "-mHKY -l40 -s0.2", newick.tree = ret.ms[3])
(ret.nucleotide &lt;- read.seqgen(ret.seqgen))
(ret.snp &lt;- read.seqgen(ret.seqgen, code.type = "SNP"))

## End(Not run)
</pre>

<hr /><div style="text-align: center;">[Package <em>phyclust</em> version 0.1-18 <a href="00Index.html">Index</a>]</div>
</body></html>
