<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta name="description" content="Phylogenetic Clustering">
  <meta name="keywords" content="phylogenetic,clustering,phyloclustering,phylogenetics,evolution,phylogeny,tree,unrooted,statistics,model-based,R,phyclust,ms,seq-gen,PAML,baseml,Hap-Clustering,EM,finite mixture model">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="../style/wcc_main.css" media="screen">
  <link rel="stylesheet" href="../style/wcc_navbar.css" media="screen">

<!-- Google Analytics Tracking. -->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-2556553-3']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>

<script src="../style/w3data.js"></script>
</head>


<body bgcolor="#f8f4e7">

<div w3-include-html="./preamble_head_nav.html"></div>
<div class="container-fluid">
  <ul class="nav nav-tabs">
    <li class=""><a href="./index.html">About phyclust</a></li>
    <li class=""><a href="./overview.html">Overview</a></li>
    <li class=""><a href="./method.html">Method</a></li>
    <li class=""><a href="./download.html">Download</a></li>
    <li class=""><a href="./document.html">Document</a></li>
    <li class="active"><a href="#">Jargon</a></li>
    <li class=""><a href="./example.html">Example</a></li>
    <li class=""><a href="./application.html">Application</a></li>
  </ul>
</div>


<xmp theme="readable" style="display:none;">

Some terminologies used in <code>phyclust</code>
will be listed in the following. The details can be found on wiki,
google, or in literatures.


---
#### Phylogenetics

- <b>Instantaneous rate matrix $\boldsymbol{Q}$</b>
  * <b>JC69</b><br>
    Jukes, T. H. and Cantor, C. R. 1969.
    "Evolution of protein molecules."
    Mammalian Protein Metabolism.  Academic Press.  <b>3</b>:21-132.

  * <b>K80</b><br>
    Kimura, M. 1980.
    "A Simple Method for Estimating Evolutionary Rates of Base Substitutions
    Through Comparative Studies of Nucleotide Sequences."
    Journal of Molecular Evolution. <b>16</b>:111--120.

  * <b>HKY85</b><br>
    Hasegawa, M. and Kishino, H. and Yano, T. 1985.
    "Dating of the human-ape splitting by a molecular clock of
     mitochondrial DNA."
    Journal of Molecular Evolution.  <b>22</b>:160--174.


- <b>Identifier (EE, EV, VE, VV) for $K$ clusters</b>
  <table>
    <thead>
      <tr align="center">
        <th bgcolor="#d3dce3"> Identifier </th>
        <th bgcolor="#d3dce3"> $\boldsymbol{Q}$ </th>
        <th bgcolor="#d3dce3"> $t$ </th>
      </tr>
    </thead>

    <tbody>
      <tr align="center">
        <td bgcolor="#dddddd"> <code>EE</code> </td>
        <td bgcolor="#dddddd"> $\boldsymbol{Q}\_1 = \boldsymbol{Q}\_2 = \cdots = \boldsymbol{Q}\_K$ </td>
        <td bgcolor="#dddddd"> $t\_1 = t\_2 = \cdots = t\_K$ </td>
      </tr>
      <tr align="center">
        <td bgcolor="#cccccc"> <code>EV</code> </td>
        <td bgcolor="#cccccc"> $\boldsymbol{Q}\_1 = \boldsymbol{Q}\_2 = \cdots = \boldsymbol{Q}\_K$ </td>
        <td bgcolor="#cccccc"> $t\_1 \neq t\_2 \neq \cdots \neq t\_K$ </td>
      </tr>
      <tr align="center">
        <td bgcolor="#dddddd"> <code>VE</code> </td>
        <td bgcolor="#dddddd"> $\boldsymbol{Q}_1 \neq \boldsymbol{Q}\_2 \neq \cdots \neq \boldsymbol{Q}\_K$ </td>
        <td bgcolor="#dddddd"> $t\_1 = t\_2 = \cdots = t\_K$ </td>
      </tr>
      <tr align="center">
        <td bgcolor="#cccccc"> <code>VV</code> </td>
        <td bgcolor="#cccccc"> $\boldsymbol{Q}_1 \neq \boldsymbol{Q}\_2 \neq \cdots \neq \boldsymbol{Q}\_K$ </td>
        <td bgcolor="#cccccc"> $t\_1 \neq t\_2 \neq \cdots \neq t\_K$ </td>
      </tr>
    </tbody>
  </table>

- <b>Neighbor-Joining Method</b>
  * Saitou, N. and Nei, M. 1987.
    "The Neighbor-Joining Method: A New Method for Reconstructing
    Phylogenetic Trees."
    Molecular Biology and Evolution.
    <b>4</b>:406--425.


---
#### Model-based Clustering

- <b>Rnd-EM</b><br>
  Maitra, R. 2007.
  "Initializing partition-optimization algorithms."
  IEEE/ACM Transactions on Computational Biology and Bioinformatics.
  <b>6</b>:114--157.

- <b>emEM</b><br>
  Biernacki, C. Celeux, G. and Govaert, G. 2003.
  "Choosing starting values for the EM algorithm for getting the
  highest likelihood in multivariate Gaussian mixture models."
  Computational Statistics and Data Analysis.
  <b>41</b>:561--575.

- <b>ECM</b><br>
  Meng, X.-L. and Rubin, D.B. 1993.
  "Maximum Likelihood Estimation via the {ECM} Algorithm: A General Framework."
  Biometrika
  <b>80</b>:267--278.

- <b>AECM</b><br>
  Meng, X.-L. and van Dyk, D. 1997.
  "The EM Algorithm --- an Old Folk-song Sung to a Fast New Tune
  (with discussion)."
  Journal of the Royal Statistical Society Series B.
  <b>59</b>:511--567.


---
<div w3-include-html="./preamble_tail_date.html"></div>

</xmp>


<script type="text/javascript" src="../style/strapdown.js"></script>
<script>
  w3IncludeHTML();
</script>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
</script>
<script type="text/javascript" src="../style/MathJax/MathJax.js?config=TeX-AMS_HTML">
</script>

</body>
</html>

